docs/learn/learn-nomination-pools.md

```diff
@@ -0,0 +1,143 @@
+---
+id: learn-nomination-pools
+title: Nomination Pools
+sidebar_label: Nomination Pools
+description: Learn about Nomination Pools and their features
+keyword: [nominate, nominator, stake, staking, pools]
+slug: ../learn-nomination-pools
+---
+
+[Nomination pools](https://github.com/paritytech/substrate/pull/10694) are one of the key
+(expertimental) features from the roadmap of
+[Staking improvements](https://gist.github.com/kianenigma/aa835946455b9a3f167821b9d05ba376) on
+Polkadot. They are designed to permissionlessly allow users (delegators) to pool their funds
+together and act as a single nominator account. Due to the current runtime constraints, Polkadot can
+only handle {{ max_active_nominator_count }} nominators comfortably in the
+[electing set](learn-nominator.md#staking-election-stages). As one of the objectives of the
+[NPoS algorithm](learn-phragmen.md) is to maximize the overall stake on the network, it can be
+inferred that the staking system on Polkadot favors nominators with a larger stake. Only the
+nomiator accounts which back the Validators in the active set are eligible for receiving staking
+rewards. This leaves out nomination intents from the accounts with lower DOT balance than the
+min-active nomination and places them in a waiting queue to enter electing set. Nomination pools
+will be handy to the users (delegators) who would like to participate the staking system with a
+stake much lower than the dynamic min-active nomination threshold on the network. All operations are
+constant space and time complexity relative to the number of delegators, eliminating any theoretical
+upper bound on the quantity of delegators the system can handle and thus scaling the number of
+accounts that can participate and earn rewards in the staking system on Polkadot.
+
+The earnings of the pool are split pro rata to a delegators stake in the bonded pool (and thus the
+staking rewards for delegators will be the same as if they were a nominator). Importantly, slashes
+are also applied proportionally to delegators who may have been actively bonded when a validator
+committed a slashable offence.
+
+## Key Components of Nomination Pools
```

Comment:
I see an explanation missing that says each pool is one nomination from the PoV of the rest of the system. This is really easy to covey with a figure


![Uploading Screen Shot 2022-04-29 at 13.12.36.png…]()